BLE_SM/mbed_app.json

```diff
@@ -0,0 +1,16 @@
+{
+    "target_overrides": {
+        "K64F": {
+            "target.features_add": ["BLE"],
+            "target.extra_labels_add": ["ST_BLUENRG"]
+        },
+        "NUCLEO_F401RE": {
```

Comment:
Please document that it is using the Cordio port for ST (as opposed to ST's official stack)

Reply:
note made about it in the readme